docs/shared/migration/from-turborepo.md

```diff
@@ -0,0 +1,189 @@
+| `pipeline[task].dotEnv`         | Define [file `inputs`](/reference/project-configuration#filesets)                                 |
+| `pipeline[task].env`            | Define [env `inputs`](/reference/project-configuration#env-variables)                             |
+| `pipeline[task].passThroughEnv` | N/A. See [Defining Environment Variables](/recipes/tips-n-tricks/define-environment-variables)    |
+| `pipeline[task].outputs`        | [Same syntax](/reference/project-configuration#outputs).                                          |
+| `pipeline[task].cache`          | Define in the [`nx.json` `cacheableOperations` property](/reference/nx-json#tasks-runner-options) |
+| `pipeline[task].inputs`         | [Same syntax](/reference/project-configuration#filesets).                                         |
+| `pipeline[task].outputMode`     | Use the [`--output-style` command line flag](/packages/nx/documents/run-many#output-style)        |
+| `pipeline[task].persistent`     | N/A.                                                                                              |
+
+## Command Equivalents
+
+|                             |                                                                                                                                                                                         |
+| --------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `turbo run test lint build` | [`nx run-many -t test lint build`](/packages/nx/documents/run-many)                                                                                                                     |
+| `--cache-dir`               | Set in [`nx.json` under `tasksRunnerOptions.default.options.cacheDirectory`](/reference/nx-json#tasks-runner-options)                                                                   |
+| `--concurrency`             | [`--parallel`](/packages/nx/documents/run-many#parallel)                                                                                                                                |
+| `--continue`                | [Use `--nx-bail`](/packages/nx/documents/run-many#nx-bail)                                                                                                                              |
+| `--cwd`                     | Available when using the [`run-commands` executor](/packages/nx/executors/run-commands#cwd)                                                                                             |
+| `--dry-run`                 | Same syntax                                                                                                                                                                             |
```

Comment:
I don't think we have `--dry-run` when running tasks, right? Just for generating stuff.

Reply:
I think --graph is closer to their dry run on tasks, albeit it does function differently